### ISSUE TITLE
Proposed improvements for Manifest Reference

### ISSUE THREAD

**Issue by jasonblais** [mattermost/mattermost-developer-documentation#52]
Propose tweaking the top portion of the manifest reference

Current:
![image](https://user-images.githubusercontent.com/13119842/33384674-a5c38694-d4f4-11e7-8157-b11840d25c2c.png)

Proposed (up for feedback):
```
Manifest Reference
===============

The plugin manifest defines the metadata required to load and present your plugin. The manifest file should be named plugin.json or plugin.yaml and placed in the top of your plugin bundle.

Example plugin.yaml:

{code}
id: com.mycompany.myplugin
name: My Plugin
description: This is my plugin. It does stuff.
backend:
    executable: myplugin
settings_schema:
    settings:
        enable_extra_thing:
            type: bool
            display_name: Enable Extra Thing
            help_text: When true, an extra thing will be enabled!
            default: false
{code}

Table of Contents
-------------------

<TOC items here>

Followed by the documentation
```

----

It might be worth reviewing the doc formatting as well. It's okay for now, but it's a little hard to follow especially with the indentation towards the end.

![image](https://user-images.githubusercontent.com/13119842/33384801-00e340e6-d4f5-11e7-9c33-37076e2b18a9.png)

**Comment by ccbrown** [comment]
0/5 on moving things around, but I agree that the indentation on the docs is pretty bad. It's definitely something I'd like better styling for.
